content/en/docs/concepts/policy/limit-range.md

```diff
@@ -9,8 +9,8 @@ weight: 10
 {{% capture overview %}}
 
 By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
-With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one Pod or Container could monopolize all of the resources. Limit Range is a policy to constrain resource  by Pod or Container in a namespace.
+With resource quotas, cluster administrators can restrict resource consumption and creation on a namespace basis.
+Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one pod or container could monopolize all available resources. A Limit Range is a policy to constrain resource consumption by Pod or Container in a namespace.
```

Comment:
Did you mean:
```suggestion
Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one pod or container could monopolize all available resources. A LimitRange is a policy to constrain resource consumption by Pod or Container in a namespace.
```

or maybe:
```suggestion
Within a namespace, a Pod or Container can consume as much CPU and memory as defined by the namespace's resource quota. There is a concern that one pod or container could monopolize all available resources. A _limit range_ is a policy to constrain resource consumption by Pod or Container in a namespace.
```
?

Reply:
Yeah, I think that's the question I have is. It's referenced several ways in the doc. I think `LimitRange`, `Limit Range`, or `limit range` would be acceptable, but I'll review the style guide you have provided.